features/support/object_model_pages/ivl/registration/ivl_confirm_your_plan_selection.rb

```diff
@@ -18,4 +18,12 @@ def self.last_name
   def self.confirm_btn
     '#btn-continue'
   end
+
+  def self.dup_enrollment_warning_1
+    'The following person currently has coverage on'
+  end
+
+  def self.dup_enrollment_warning_2
+    'Enrolling in a new plan will remove existing coverage for these persons'
```

Comment:
let's use a `data-cuke="something"` selector for all new page object model selectors

additionally, we should not be asserting on hard-coded english text, so let's find an element to assert on